examples/acquiring_data/downloading_cutouts.py

```diff
@@ -78,37 +59,16 @@
 # We request one image every 2 hours.
 
 query = Fido.search(
-    a.Time(cutouts.date - 6*u.h, cutouts.date + 6*u.h),
-    a.Wavelength(cutouts.wavelength),
+    a.Time(amap.date - 6*u.h, amap.date + 6*u.h),
+    a.Wavelength(amap.wavelength),
     a.Sample(2*u.h),
     a.jsoc.Series.aia_lev1_euv_12s,
     a.jsoc.Notify(jsoc_email),
     a.jsoc.Segment.image,
     cutout,
 )
+print(query)
 
 #####################################################
-# Submit the export request and download the data.
-
-files = Fido.fetch(query)
-files.sort()
-
-#####################################################
-# Now that we've downloaded the files, we can create
-# a `~sunpy.map.MapSequence` from them.
-
-sequence = sunpy.map.Map(files, sequence=True)
-
-#####################################################
-# Finally, we can construct an animation in time from
-# our stack of cutouts and interactively flip through
-# each image in our sequence. We first adjust the plot
-# settings on each image to ensure the colorbar is the
-# same at each time step.
-
-for each_map in sequence:
-    each_map.plot_settings['norm'] = ImageNormalize(vmin=0, vmax=5e3, stretch=SqrtStretch())
-plt.figure()
-ani = sequence.plot()
-
-plt.show()
```

Comment:
The only reason I am sad about removing this is that this was a nice little gif on the gallery page, and much more eye catching than the now complete lack of a preview image.

Reply:
FWIW, this code can now be greatly simplified to just set `norm=...` in `sequence.plot()`.

Reply:
Is that a blocking request for this PR? I'd advocate for merge this as it's an improvement and this plotting code isn't changed, and we can sort of all the norms (if there are more?) in another PR.

Reply:
Nope not blocking. I should've cleaned these up a long time ago.